tokens_definition_example_draft_v2.json

```diff
@@ -0,0 +1,205 @@
+{
+	"tokenCategories" : [
+		{
+			"label": "general",
+			"name": "general"
+		},
+		{
+			"label": "layout",
+			"name": "layout"
+		}
+	],
+	"tokenSets" : [
+		{
+			"label": "grid",
+			"name": "grid"
+		},
+		{
+			"label": "links",
+			"name": "links"
+		}
+	],
+	"tokens": [
+		{
+			"mappings" : [
+				{
+					"name": "CSSVariable",
+					"outputKey": "$link-color",
+					"outputValues": [
+						{
+							"tokenValue": "Red",
+							"mappedValue": "#FF0000"
+						},
+						{
+							"tokenValue": "Salmon",
+							"mappedValue": "#FA8072"
+						}
+					]
+				}
+			],
+			"editorType": "ColorPicker",
+			"label": "link-color",
+			"name": "linkColor",
+			"validValues": [
+				{
+					"label": "salmon",
+					"value" : "Salmon"
+				},
+				{
+					"label": "red",
+					"value" : "Red"
+				}
+			],
+			"tokenCategoryName":  "general",
+			"tokenSetName": "links",
+			"type": "Color"
```

Comment:
I'm confused about this... I was assuming we added `editorType` because we wouldn't have `type`. Having `type: color`,  and `editorType: colorPicker` seems rather redundant information. I'd expect either we treat all as `strings` and drop `type` or use `type` as the "picker" hint.

What's the use case in which we will need both? What does "type: Color" mean in this context? Color might be represented in different ways (all strings) like color-name, rgb, hex... 

Just looking to simplify as much as we can to avoid running into ambiguous combinations of values.

Reply:
My bad. This is a mistake. It should say `"type": "String"` so it validates against the JSON Shema.

Reply:
To summarize this aspect again:

Attribute `type` would refer to possible primitive values: `Boolean`, `Integer`, `Number`, `String`
Attribute `editorType` would include information on how to to edit the value of the token (for example `ColorPicker`)

Reply:
Again, as per my [previous comment](https://github.com/liferay-frontend/liferay-portal/pull/99#discussion_r452812397), we are merging two layers of abstraction in one single file:

1. how the data type looks (token definition)
2. how the data is edited (token value)

Unless we consider `editorType` a hint to be consumed by the layer showing the token value edition...